server/server.js

```diff
@@ -92,10 +92,15 @@ module.exports = async ({graphQLConfig, graphiqlConfig, postgresConfig, schemaLi
 }
 
 async function buildSchema (models) {
-  const graphQLSchema = await models.GraphQLSchema.findOne()
+  const graphQLSchemas = await models.GraphQLSchema.findAll()
   let graphQLSchemaString = null
-  if (graphQLSchema != null) {
-    graphQLSchemaString = graphQLSchema.schema
+
+  if (!_.isEmpty(graphQLSchemas)) {
```

Comment:
What about only loading the schema with the name `default`? @pb82 how is this handled in the UI?

Reply:
@darahayes Yes, thats what the UI will do: https://github.com/aerogear/data-sync-ui/pull/31/files#diff-ca481b05655823ba99892d2d34415a03R140